xcube/core/timeseries.py

```diff
@@ -59,7 +59,7 @@
     AGG_COUNT: CAN_COMPUTE
 }
 
-_CRS84 = pyproj.CRS.from_string('CRS84')
+_CRS84 = pyproj.CRS.from_string("OGC:CRS84")
```

Comment:
We usually use single quotes for consitency, as far as possible. However, as you can tell from older code - we are mixing double quotes and single quotes... But would be great not to change existing single quotes into double quotes :)